### PR TITLE
removed unused code line evoked_nf_name

### PR DESCRIPTION
**Pull request by ARudiuk**
Removed unused code line, `evoked_nf_name` in some test files.

### CODE REVIEW

**Comment by ARudiuk** [comment]
#3271

File: mne/channels/tests/test_interpolation.py [hunk 1]
```diff
@@ -12,7 +12,6 @@
 base_dir = op.join(op.dirname(__file__), '..', '..', 'io', 'tests', 'data')
 raw_fname = op.join(base_dir, 'test_raw.fif')
 event_name = op.join(base_dir, 'test-eve.fif')
-evoked_nf_name = op.join(base_dir, 'test-nf-ave.fif')
```
Comment (agramfort) [inline mne/channels/tests/test_interpolation.py]: @EricGLX89 any idea what this file was for?

Comment (Eric89GXL) [inline mne/channels/tests/test_interpolation.py]: No filter maybe? You could git grep for where it's still used after this PR
and it would indicate it.

**Comment by ARudiuk** [comment]
It seems to only be used here

https://github.com/mne-tools/mne-python/blob/5fdb8b0c6795900e022e449596617d2f2560ebc6/mne/tests/test_epochs.py#L55

https://github.com/mne-tools/mne-python/blob/5fdb8b0c6795900e022e449596617d2f2560ebc6/mne/tests/test_epochs.py#L1033-L1048

**Comment by Eric89GXL** [comment]
Looks like a comparison to the C code computation on our test data files

**Comment by agramfort** [comment]
thanks @ARudiuk !

**Pull request merged**